### PR TITLE
[macOS] Add Homebrew installation instructions.

### PR DESCRIPTION
**Pull request by nevack**

### CODE REVIEW

**Review by pietryszak — APPROVED** [review]
Looks OK. 
Same as [brew website instruction]( https://formulae.brew.sh/formula/btop#default).

File: README.md [hunk 1]
```diff
@@ -275,6 +275,13 @@ Also needs a UTF8 locale and a font that covers:
   sudo zypper in btop
   ```
 
+**Binary release on macOS**
```
Comment (XuehaiPan) [inline README.md:278]:
```suggestion
**Binary release on Homebrew**
```

The formula in Homebrew supports both macOS (x86_64 and ARM64) and x86_64 Linux.

Comment (aristocratos) [reply README.md:278]: Maybe `Homebrew (macos, linux)` to really make clear what the supported platforms are.

**Review by aristocratos — APPROVED** [review]
Awesome, thanks again for creating the formula!

**Pull request merged**